cob_default_robot_config/cob3-2/base_configurations.yaml

```diff
@@ -0,0 +1,2 @@
+service_ns: /base_controller
```

Comment:
why? I don't now who did it, but the cob3-bases are using the old driver , and the old ns like cob4-2: https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_default_robot_config/cob4-2/base_configurations.yaml